lib/cicognara/catalogo_item.rb

```diff
@@ -2,8 +2,9 @@
 class CatalogoItem
   attr_accessor :xml_element
 
-  def initialize(xml_element)
+  def initialize(xml_element, xsl)
```

Comment:
Not going to argue about it, but does the XSL need to be parameterized? Can't it be encapsulated here or in the `TEIIndexer`?